apps/settings/style/downloads.css

```diff
@@ -93,20 +93,17 @@
 
 /* Text */
 .fileName {
-  padding-top: 1rem;
-  -moz-padding-end: 6rem;
-  padding-bottom: 1rem;
-  -moz-padding-start: 3.5rem;
-  -moz-box-sizing: border-box;
+  padding: 1rem 6rem;
+  box-sizing: border-box;
```

Comment:
oh. No. This has been replaced by the two lines below:
```css
html[dir="ltr"] .fileName { padding-left: 3.5rem; }
html[dir="rtl"] .fileName { padding-right: 3.5rem; }
```

These two lines work even if the `.fileName` element has a `dir="auto"` attribute — and it needs one to display LTR file names in an RTL context.